tailwind.config.js

```diff
@@ -133,5 +133,6 @@ module.exports = {
   plugins: [
     require('@tailwindcss/forms'),
     require('@tailwindcss/line-clamp'),
-  ]
+  ],
+  important: true
```

Comment:
I don't think this is a good idea. Can we instead style react-tooltip directly with normal CSS instead of using tailwind?

Reply:
Will do

Reply:
Neat, tailwind has an `@apply` that lets you just throw tailwind classes into your normal CSS.

```
.light-tooltip {
  @apply rounded-md shadow-lg bg-white ring-1 ring-black ring-opacity-5 opacity-100 p-4 !important;
}
```